templates/guides/get-started.md

```diff
@@ -0,0 +1,216 @@
+
+## Step 3: Authenticate with Astro
+1. [Create an API token](https://www.astronomer.io/docs/astro/automation-authentication#step-1-create-an-api-token) in Astro. Since you are creating a Workspace, you need an [Organization API token](https://www.astronomer.io/docs/astro/organization-api-tokens) with [Organization Owner permissions](https://www.astronomer.io/docs/astro/user-permissions#organization-roles).
+2. Configure the API token as an environment variable `ASTRO_API_TOKEN` to run Terraform commands:
+`export ASTRO_API_TOKEN=<your-api-token>`
+
+Alternatively, users can set their API token value in the provider block:
+```
+provider "astro" {
+  organization_id = "&lt;your-organization-id&gt;"
+  token = "&lt;your-api-token&gt;"
+}
+```
+
+## Step 4: Define Resources in Terraform
+In a file `main.tf`, define two resources, an `astro_workspace` and an `astro_deployment`. These resources will represent an Astro workspace and Astro Deployment, defined in Terraform code:
+```
+# Create a new workspace
+resource "astro_workspace" "my_first_tf_workspace" {
+  name                  = "My first TF workspace"
+  description           = "My first Terraform-created workspace"
+  cicd_enforced_default = false
+}
+
+# Create a new Deployment
+resource "astro_deployment" "my_first_tf_deployment" {
+  name                    = "My first TF deployment"
+  description             = "My first Terraform-created deployment"
+  type                    = "STANDARD"
+  cloud_provider          = "AWS"
+  region                  = "us-east-1"
+  contact_emails          = []
+  default_task_pod_cpu    = "0.25"
+  default_task_pod_memory = "0.5Gi"
+  executor                = "CELERY"
+  is_cicd_enforced        = true
+  is_dag_deploy_enabled   = true
+  is_development_mode     = false
+  is_high_availability    = false
+  resource_quota_cpu      = "10"
+  resource_quota_memory   = "20Gi"
+  scheduler_size          = "SMALL"
+  workspace_id            = astro_workspace.my_first_tf_workspace.id
+  environment_variables   = []
+  worker_queues = [{
+    name               = "default"
+    is_default         = true
+    astro_machine      = "A5"
+    max_worker_count   = 10
+    min_worker_count   = 0
+    worker_concurrency = 1
+  }]
+}
+```
+-> One of the key characteristics (and benefits) of using Terraform is that it's *declarative*. For example, `workspace_id = astro_workspace.my_first_tf_workspace.id` tells Terraform to configure the Workspace ID in the Deployment. This means the Workspace must be created first, producing an ID which is a generated value and unknown at the time of writing. You don't have to instruct Terraform to create resources in a certain order, you only have to instruct what to create. The resources above can be defined in any order. Terraform takes the relationships between resources into account when deciding the order of creating resources.
```

Comment:
> Unknown at the time of writing

Who is writing? This sentence might need to get the wording tweaked.

Reply:
Updated wording to
"This means the Workspace must be created first, producing an ID which is a generated value and unknown at the time you initially define the resources in Terraform."